packages/plugins/Web3Profile/src/SNSAdaptor/components/FollowLensDialog.tsx

```diff
@@ -0,0 +1,287 @@
+import { Icons } from '@masknet/icons'
+import { InjectedDialog, WalletConnectedBoundary } from '@masknet/shared'
+import { CrossIsolationMessages, NetworkPluginID } from '@masknet/shared-base'
+import { useRemoteControlledDialog } from '@masknet/shared-base-ui'
+import { ActionButton, makeStyles } from '@masknet/theme'
+import { useChainContext, useFungibleTokenBalance, useWallet } from '@masknet/web3-hooks-base'
+import { Lens } from '@masknet/web3-providers'
+import { FollowModuleType } from '@masknet/web3-providers/types'
+import { formatBalance, isLessThan } from '@masknet/web3-shared-base'
+import { ChainId } from '@masknet/web3-shared-evm'
+import { Avatar, Box, Button, buttonClasses, CircularProgress, DialogContent, Typography } from '@mui/material'
+import { first } from 'lodash-es'
+import { useMemo, useState } from 'react'
+import { useAsyncRetry, useHover } from 'react-use'
+import { Translate, useI18N } from '../../locales/i18n_generated.js'
+import { getLensterLink } from '../../utils.js'
+import { useFollow } from '../hooks/Lens/useFollow.js'
+import { useUnfollow } from '../hooks/Lens/useUnfollow.js'
+import { HandlerDescription } from './HandlerDescription.js'
+
+const useStyles = makeStyles()((theme) => ({
+    container: {
+        display: 'flex',
+        flexDirection: 'column',
+        alignItems: 'center',
+    },
+    name: {
+        fontSize: 16,
+        lineHeight: '20px',
+        fontWeight: 700,
+        color: theme.palette.maskColor.main,
+        marginTop: 24,
+    },
+    handle: {
+        fontSize: 16,
+        lineHeight: '20px',
+        fontWeight: 400,
+        color: theme.palette.maskColor.main,
+        margin: theme.spacing(0.5, 0),
+    },
+    followers: {
+        display: 'flex',
+        justifyContent: 'center',
+        columnGap: 8,
+    },
+    dialogTitle: {
+        background: `${theme.palette.maskColor.bottom}!important`,
+    },
+    dialogContent: {
+        maxWidth: 400,
+        minHeight: 398,
+    },
+    actions: {
+        display: 'flex',
+        flexDirection: 'column',
+        rowGap: 12,
+        width: '100%',
+        marginTop: 24,
+    },
+    followAction: {
+        backgroundColor: '#A1FE27',
```

Comment:
Use MaskColor.

Reply:
This color doesn't belong to the theme color.